spec/typinggame_server/interactors/players/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Interactors::Players::Index do
+  let(:repository) { instance_double(PlayersRepository) }
```

Comment:
I'm not a big fan of stubbing out the repository (or anything really) if we can get away with it.

Reply:
Is this something Hanami recommends?

Reply:
@jgayfer No, I found this way of writing an interactor spec online, I'll look into writing it differently to test actual code.

Reply:
@jgayfer fixed.